.travis.yml

```diff
@@ -26,58 +30,69 @@ cache:
   - .stack-work
 
 install:
-  # 1. Download and unpack the stack executable
+  # 1. Download, unpack, and check the stack executable
   - mkdir -p ~/.local/bin
   - export PATH=$HOME/.local/bin:$PATH
-  - travis_retry curl -L https://github.com/commercialhaskell/stack/releases/download/v1.9.3/stack-1.9.3-linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
-  # 2. Configure stack to use the system GHC installation
-  - stack config set system-ghc --global true
-  - export PATH=/opt/ghc/ghc-8.6.4/bin:$PATH
-  # 3. Init Application Config
+  - travis_retry curl -L https://get.haskellstack.org/stable/linux-x86_64.tar.gz | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
+  - stack --version
+  # 2. Init Application Config
   - cp config/app-config.cfg.example config/app-config.cfg
   - cp config/app-config-test.cfg.example config/app-config-test.cfg
 
 before_script:
-  # 1. Log to both public and our private Docker Registry
+  # 1. Log to public Docker Hub
   - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
+  # 2. Log to private Docker Registry
   - docker login -u "$REGISTRY_USERNAME" -p "$REGISTRY_PASSWORD" "$REGISTRY"
 
 jobs:
   include:
     - stage: build
       script:
-        # 1. Init testing Mongo Database
+        # 1. Init & check testing Mongo Database
```

Comment:
I would write `Init and check testing Mongo Database`

Reply:
Fixed in https://github.com/ds-wizard/dsw-server/pull/61/commits/8f1371210d990a2652c07e43231558be432ace83